### PR TITLE
Add Thomas Robitaille as fits maintainer

### PR DESCRIPTION
**Pull request by eteq**
Just what it says on the tin!  Note this is a follow-up of #309, as @astrofrog realized there that he was willing to take on this role

### CODE REVIEW

File: roles.json [hunk 1]
```diff
@@ -434,6 +434,7 @@
             {
                 "role": "astropy.io.fits",
                 "people": [
+                    "Thomas Robitaille",
```
Comment (pllim) [inline roles.json:437]: What does this order mean and is Simon okay with it?

Comment (mhvk) [reply roles.json:437]: Indeed, might as well keep order alphabetical as is the case for all the other subpackages.

**Review by pllim — APPROVED** [review]
It's been over 2 weeks comment period and no objection. Be careful what you ask for, @astrofrog ! 😸

File: roles.json (hunk 1 above)
Comment (pllim) [reply roles.json]: I fixed it.

**Comment by pllim** [comment]
He is already a core maintainer, no follow-up needed.

**Pull request merged**